### DISCUSSION TITLE
PLS Removing Pre-Release Notifications

### DISCUSSION THREAD

**Discussion by ffxivcookiekitten** [uBlockOrigin/uBlock-issues#3094] [General] [answered]
title says it. there are annyoing. just notification on full release.

**Comment by gorhill** [comment, marked as answer]
That is not under my control. Just unsubscribe from these notifications on your side.

![image](https://github.com/uBlockOrigin/uBlock-issues/assets/585534/2d693df3-f2a2-4f3b-a5f7-9df321c7794a)
